tests/ceph_ansible/shrink_mon.py

```diff
@@ -37,17 +38,42 @@ def run(ceph_cluster, **kw):
     ceph_installer.node.obtain_root_permissions('/var/log')
     ansible_dir = ceph_installer.ansible_dir
 
-    ceph_installer.exec_command(sudo=True, cmd='cp -R {ansible_dir}/infrastructure-playbooks/shrink-mon.yml '
-                                               '{ansible_dir}/shrink-mon.yml'.format(ansible_dir=ansible_dir))
+    out1, rc = ceph_installer.exec_command(sudo=True, cmd='cd {ansible_dir};cat hosts'.format(ansible_dir=ansible_dir))
+    outbuf = out1.read().decode()
+    inventory = outbuf.split("\n")
+    log.info("\nInventory before shrink-mon playbook\n")
+    log.info(outbuf)
+    # Set ansible deprecation warning to false
+    ceph_installer.exec_command(cmd='export ANSIBLE_DEPRECATION_WARNINGS=False')
 
-    out, err = ceph_installer.exec_command(cmd='export ANSIBLE_DEPRECATION_WARNINGS=False ; cd {ansible_dir} ; '
-                                               'ansible-playbook -vvvv -e ireallymeanit=yes shrink-mon.yml '
-                                               '-e mon_to_kill={mon_to_kill} -i hosts'.format(ansible_dir=ansible_dir,
-                                                                                              mon_to_kill=mon_to_kill),
-                                           long_running=True)
+    if build.startswith('4'):
+        cmd = 'cd {ansible_dir} ;ansible-playbook -vvvv -e \
```

Comment:
Would be good if we make common command string and update using if else block.

for example:
```
cmd = "ansible-playbook -i hosts {playbook}"

if condition1:
    playbook = "infra/site1.yml"
else:
    playbook = "site.yml"

client.exec_command(cmd=cmd.format(playbook=playbook))
```